doc/source/complex_composites.rst

```diff
@@ -0,0 +1,120 @@
+Complex Composites
+==================
+
+Many composites in |project| take advantage of various corrections or
+adjustments to produce the best looking imagery possible. The below
+sections describe the corrections and other related topics used in
+|project|. See the various :doc:`readers/index` documentation for more
+information on what products are available and descriptions of what
+corrections are used.
+
+.. _explain_rgb_composite:
+
+RGB Images
+----------
+
+An RGB image is an image constructed of three different color channels,
+Red, Green, and Blue. By interpreting the values from each channel and
+combining them we can produce an array of colors. This differs from a
+basic grayscale or "luminance" image where the single "L" channel
+determines how white or black a pixel is. For more information on the
+RGB color model see
+`this Wikipedia article <https://en.wikipedia.org/wiki/RGB_color_model>`_.
+
+You may also see RGBA images where the A stands for "Alpha". This Alpha
+channel will determine the opaque or transparency of an image. This is
+typically used in |project| to make invalid or missing data values
+transparent (completely opaque or completely transparent).
+
+.. _sunz_correction:
+
+Solar Zenith Angle Modification
+-------------------------------
+
+Visible channel data is not truly a "reflectance" until it has been adjusted
+for the amount of sun reflecting off of the atmosphere. While this is a very
+common change when working with numeric reflectance data, some RGB recipes
+may not include it.
```

Comment:
Does this mean some RGB recipes used by G2G, or other software?

Reply:
Other software. For example the CIMSS True Color recipe does not apply this adjustment.

Reply:
To clarify, as far as I know there are no default RGBs in G2G that don't apply this adjustment to reflectance channels.

Reply:
If I understand it correctly, how about something like this for the whole section:
Reflectance is defined as the reflected radiation as a fraction of the incident radiation. To calculate reflectance, the solar zenith angle is needed, in addition to the radiance measured by the sensor. Some of the RGB recipes applied by Geo2Grid use reflectance (with the solar zenith angle modification), and others use radiance directly.

Reply:
Oh well now by using "radiance" in that last part you've opened up a can of worms. Geo2Grid applies various calibration steps to convert the digital numbers in the file to a "normalized radiance" (at least I think that's what @kathys calls it). This is different than the radiance that you can also convert these digital numbers to. This issue has resulted in a huge discussion in the pytroll community, with Kathy, and with the GOES-R Imagery team. See https://github.com/pytroll/satpy/issues/536# for some of the pytroll discussion.

Reply:
Happily, you can scratch the last sentence because I wrote it before I read your comment and it sounds like it is not needed anyway.

Reply:
Just realized the other important part here is that the individual channel images (C01-C06) don't have this applied.